Automattic/wp-calypso
# Add/new email comparison card workspace wire

#### Changes proposed in this Pull Request

The changes in this pull request are allowing the Google Workspace Card to add the product to the shopping cart.

#### Testing instructions

1. Go to a site with domains without email subscription
2. Click on 'Add email' for the desired site
3. Once you are in the comparison page, add at the end the following flag: `?flags=emails/new-email-comparison`
4. Try to fill form data and check that the item is added into the cart and redirected to the checkout.

![image](https://user-images.githubusercontent.com/5689927/144414474-3cf3b6ef-b04f-4f3b-9101-f90f7e7d02c5.png)

Related to {1200182182542585-as-1201457616446698}
